src/ansible_navigator/configuration_subsystem/navigator_post_processor.py

```diff
@@ -281,6 +281,22 @@ def execution_environment(self, entry: SettingsEntry, config) -> PostProcessorRe
                         )
                     if hint:
                         exit_messages.append(ExitMessage(message=hint, prefix=ExitPrefix.HINT))
+
+            # check for /dev/mqueue/ when using podman because runner passes ipc=host
+            # https://github.com/ansible/ansible-navigator/issues/610
+            mqueue_path = "/dev/mqueue/"
+            if config.container_engine == "podman":
+                if not Path(mqueue_path).exists() or not Path(mqueue_path).is_dir():
```

Comment:
Plese combine these clauses to dedent the whole conditional block:
```suggestion
            podman_engine_with_no_mqueue_path = (
                config.container_engine == "podman" and
                not Path(mqueue_path).exists() or not Path(mqueue_path).is_dir()
            (
            if podman_engine_with_no_mqueue_path:
```
This way, there's a named conditional with a clearer purpose/semantics + less nesting.

Reply:
I actually like the original here better because it's easier than having to guess which way Python will associate the set of conditionals. If you do take @webknjaz's suggestion, please wrap the second set of conditionals in parens so it's easy to see at a glance what the actual intent is for this to be parsed.

Reply:
Also, could just name that second conditional too.

Reply:
Looking closer, I realized that there's `Path(mqueue_path).exists()` before `is_dir()` — why is it even necessary? Wouldn't `is_dir()` only be true if it exists?

Reply:
yep! kinda funny :)

Reply:
@relrod  let me know if I captured your feedback. A lot changed, but I think I got it...

Reply:
Be warned that is_dir will returns false for symlinked folders. They act as folders but is_dir reports false.